plugins/gaming.py

```diff
@@ -15,32 +16,30 @@
 
 from cloudbot import hook
 
+# String constants for the coin flip function
+INVALID_NUMBER = "Invalid input '{}': not a number"
+NO_COIN = "makes a coin flipping motion"
+SINGLE_COIN = "flips a coin and gets {}."
+MANY_COINS = "flips {} coins and gets {} heads and {} tails."
+
+ROLL_LIMIT = 100  # The maximum number of times to roll or flip before approximating results
+
 whitespace_re = re.compile(r'\s+')
 valid_diceroll = re.compile(r'^([+-]?(?:\d+|\d*d(?:\d+|F))(?:[+-](?:\d+|\d*d(?:\d+|F)))*)( .+)?$', re.I)
 sign_re = re.compile(r'[+-]?(?:\d*d)?(?:\d+|F)', re.I)
 split_re = re.compile(r'([\d+-]*)d?(F|\d*)', re.I)
 
 
-def clamp(n, min_value, max_value):
-    """Restricts a number to a certain range of values,
-    returning the min or max value if the value is too small or large, respectively
-    :param n: The value to clamp
-    :param min_value: The minimum possible value
-    :param max_value: The maximum possible value
-    :return: The clamped value
-    """
-    return min(max(n, min_value), max_value)
-
-
 def n_rolls(count, n):
     """roll an n-sided die count times
+
     :type count: int
     :type n: int | str
     """
     if n in ('f', 'F'):
-        return [random.randint(-1, 1) for _ in range(min(count, 100))]
+        return [random.randint(-1, 1) for _ in range(min(count, ROLL_LIMIT))]
```

Comment:
Might want to change this to "fake" the result similar to the other checks, for consistency.

Reply:
I will look into finding a method to approximate the results. The method for other rolls will not work directly with the fudge dice because of the nature of the dice.